tests/unit/authentication_test.py

```diff
@@ -6,7 +6,7 @@
 from bond_app.authentication import Authentication, AuthenticationConfig, UserInfo
 from bond_app.sam_api import SamApi, SamKeys
 from tests.unit.fake_cache_api import FakeCacheApi
-import json
+import time
 
 
 class TestRequestState:
```

Comment:
Refactored this test quite a bit based on new logic. Added comments to each test case.